src/simulator.ts

```diff
@@ -88,13 +89,51 @@ export class Simulator {
 
   public async cancel() {
     this.cancelToken = true;
+    await this.delay(1000);
+  }
+
+  public async telemetry(eventName: string, status: boolean, properties?: { [key: string]: string; }) {
+    const iotHubConnectionString = status ? await Utility.getConnectionString(
+      Constants.IotHubConnectionStringKey,
+      Constants.IotHubConnectionStringTitle,
+      false,
+    ) : undefined;
+    if (eventName === Constants.SimulatorLaunchEvent) {
+      TelemetryClient.sendEvent(eventName, {
```

Comment:
1. status->Result, and the values should be Fail/Success. So we can align with exiting telemetries.
2. reason->Error, since we only record errors here
3. Property names start with capital character to align with existing naming style

Reply:
Also rename the status/reason in other places to keep the property name consistent

Reply:
Name changed.